pkg/config/configmap.go

```diff
@@ -68,4 +73,7 @@ type Kourier struct {
 	EnableServiceAccessLogging bool
 	// EnableProxyProtocol specifies whether proxy protocol feature is enabled
 	EnableProxyProtocol bool
+	// KourierInternalCertSecret specifies the secret name for the server certificates of
+	// Kourier Internal.
+	KourierInternalCertSecret string
```

Comment:
Again, I'd call this `ClusterCertSecret`, so as not to repeat ("stutter") `Kourier`, and to be more clear that this is a cert that clients in the cluster will see.